packages/app/src/components/Layout/__tests__/TooltipContent.spec.js

```diff
@@ -1,33 +1,26 @@
 import React from 'react'
 import { shallow } from 'enzyme'
-import Popper from '@material-ui/core/Popper'
-import { Tooltip } from '../Tooltip'
+import { TooltipContent } from '../TooltipContent'
 
-describe('Tooltip', () => {
+describe('TooltipContent', () => {
     let props
     let shallowTooltip
     const tooltip = () => {
         if (!shallowTooltip) {
-            shallowTooltip = shallow(<Tooltip {...props} />)
+            shallowTooltip = shallow(<TooltipContent {...props} />)
         }
         return shallowTooltip
     }
 
     beforeEach(() => {
         props = {
-            open: true,
-            anchorEl: {},
             dimensionId: 'abc',
             metadata: {},
             itemIds: [],
         }
         shallowTooltip = undefined
     })
 
-    it('renders a Popper', () => {
```

Comment:
This could be rewritten to "renders a UL" and reused again.

Reply:
Done.